### PR TITLE
Fix Devise missing translations error

### PR DESCRIPTION
**Pull request by dinocosta**
- Fix missing pt translations for Devise.
- Add rendering of notices in user profile page so it displays
the successful login message.

### CODE REVIEW

**Comment by dinocosta** [comment]
@frmendes Should I wait for your approval or just close this? :)

File: config/locales/devise.pt.yml
Comment (frm) [inline config/locales/devise.pt.yml]: why pt?

**Review by frm — CHANGES_REQUESTED** [review]

File: config/locales/devise.pt.yml
Comment (dinocosta) [reply config/locales/devise.pt.yml]: @frmendes Because the i18 default locale is set to `:pt` in `config/application.rb`. Should we change it to `:en` and just roll with it?

Comment (frm) [reply config/locales/devise.pt.yml]: Change it to `en` and change the default locale. The site's in english either way.

Comment (dinocosta) [reply config/locales/devise.pt.yml]: @frmendes I agree! Updated it, can you review it? 👍

**Review by frm — APPROVED** [review]

**Pull request merged**
